student/src/main/webapp/resources/sections/TTSCheck.xhtml

```diff
@@ -1,122 +1,186 @@
+						class="checker soundReady"
+						i18n-content="TTSCheck.Label.ClickToHear"><span></span></a>
+
 					<div lang="en-us" id="englishMessageDiv">
-						<span lang="en-us" style="display: none" i18n-content="TTSCheck.Label.HearMessageEnglish"></span>
+						<span lang="en-us" style="display: none"
+							i18n-content="TTSCheck.Label.HearMessageEnglish"></span>
 					</div>
 					<p i18n-content="TTSCheck.Label.HeardYesOrNo"></p>
 				</div>
-				<span class="clear"></span> <span class="actions"> <tds:Button id="btnSoundEnglishNo" styleClass="negative" key="TTSCheck.Label.HeardNo" /> <tds:Button id="btnSoundEnglishYes"
-						styleClass="positive" key="TTSCheck.Label.HeardYes" /> <tds:Button id="skipEnglish" styleClass="positive" key="TTSCheck.Label.SkipEnglish" />
-				</span> <span class="clear"></span>
+				<span class="clear"></span>
+				<tds:NextStepWrapper headerID="sectionCheckSoundEnglishNextSteps"
+					headerName="CheckSoundEnglish">
+					<tds:Button id="btnSoundEnglishYes" styleClass="positive"
+						key="TTSCheck.Label.HeardYes" />
+					<tds:Button id="btnSoundEnglishNo" styleClass="negative"
+						key="TTSCheck.Label.HeardNo" />
+					<tds:Button id="skipEnglish" styleClass="positive"
+						key="TTSCheck.Label.SkipEnglish" />
+				</tds:NextStepWrapper>
+				<span class="clear"></span>
 			</div>
 		</div>
 
 		<div id="spanishTTSTest" class="hidden">
 			<div id="checkSoundSpanish">
-				<span i18n-content="TTSCheck.Label.TTSSpanish"></span>
-				<h1 i18n-content="TTSCheck.Label.DidYouHearSpanish"></h1>
+				<h1 i18n-content="Sections.TopHeader.CheckSoundSpanish"></h1>
+				<div class="instructions-top"
+					i18n-content="Sections.TopInstructions.CheckSoundSpanish"></div>
 				<div class="instructions">
-					<p>
-						<a id="clickToHearSpanish" href="javascript:void(0);" class="checker soundReady" tabindex="1" i18n-content="TTSCheck.Label.ClickToHearSpanish"><span></span></a>
-					</p>
+
+					<a id="clickToHearSpanish" href="javascript:void(0);"
+						class="checker soundReady" tabindex="1"
+						i18n-content="TTSCheck.Label.ClickToHearSpanish"><span></span></a>
+
 					<div lang="es-mx" id="spanishMessageDiv">
-						<span lang="es-mx" style="display: none" i18n-content="TTSCheck.Label.HearMessageSpanish"></span>
+						<span lang="es-mx" style="display: none"
+							i18n-content="TTSCheck.Label.HearMessageSpanish"></span>
 					</div>
 					<p i18n-content="TTSCheck.Label.HeardYesOrNoSpanish"></p>
 				</div>
-				<span class="clear"></span> <span class="actions"> <tds:Button id="btnSoundSpanishNo" styleClass="negative" key="TTSCheck.Label.HeardNoSpanish" /> <tds:Button id="btnSoundSpanishYes"
-						styleClass="positive" key="TTSCheck.Label.HeardYesSpanish" /> <tds:Button id="skipSpanish" styleClass="positive" key="TTSCheck.Label.SkipSpanish" />
-				</span> <span class="clear"></span>
+				<span class="clear"></span>
+				<tds:NextStepWrapper id="sectionCheckSoundSpanishNextSteps"

```

Comment:
Are there changes here or is this just formatting? Please submit a clearer diff.